tools/pycbc_test_suite.sh

```diff
@@ -11,77 +11,63 @@ LOG_FILE=$(mktemp -t pycbc-test-log.XXXXXXXXXX)
 
 RESULT=0
 
+function test_result {
+    if test $? -ne 0 ; then
+        RESULT=1
+        echo -e "    FAILED!"
+        echo -e "---------------------------------------------------------"
+        cat $LOG_FILE
+        echo -e "---------------------------------------------------------"
+    else
+        echo -e "    Pass"
+    fi
+}
+
 if [ "$PYCBC_TEST_TYPE" = "unittest" ] || [ -z ${PYCBC_TEST_TYPE+x} ]; then
     for prog in `find test -name '*.py' -print | egrep -v '(long|lalsim|test_waveform)'`
     do
         echo -e ">> [`date`] running unit test for $prog"
         python $prog &> $LOG_FILE
-        if test $? -ne 0 ; then
-            RESULT=1
-            echo -e "    FAILED!"
-            echo -e "---------------------------------------------------------"
-            cat $LOG_FILE
-            echo -e "---------------------------------------------------------"
-        else
-            echo -e "    Pass."
-        fi
+	test_result
     done
 fi
 
 if [ "$PYCBC_TEST_TYPE" = "help" ] || [ -z ${PYCBC_TEST_TYPE+x} ]; then
     # check that all executables that do not require
     # special environments can return a help message
-    for prog in `find ${PATH//:/ } -maxdepth 1 -name 'pycbc*' -print 2>/dev/null | egrep -v '(pycbc_live_nagios_monitor|pycbc_make_offline_grb_workflow|pycbc_mvsc_get_features|pycbc_upload_xml_to_gracedb|pycbc_coinc_time)' | sort | uniq`
+    for prog in `find ${PATH//:/ } -maxdepth 1 -name 'pycbc*' -print 2>/dev/null | egrep -v '(pycbc_live_nagios_monitor|pycbc_mvsc_get_features|pycbc_coinc_time)' | sort | uniq`
     do
         echo -e ">> [`date`] running $prog --help"
         $prog --help &> $LOG_FILE
-        if test $? -ne 0 ; then
-            RESULT=1
-            echo -e "    FAILED!"
-            echo -e "---------------------------------------------------------"
-            cat $LOG_FILE
-            echo -e "---------------------------------------------------------"
-        else
-            echo -e "    Pass."
-        fi
+	test_result
+	if [[ `echo $prog | egrep '(pycbc_copy_output_map|pycbc_submit_dax|pycbc_stageout_failed_workflow)'` ]] ; then
+	    continue
+	fi
+	echo -e ">> [`date`] running $prog --version"
+	$prog --version &> $LOG_FILE
+	test_result
```

Comment:
The indentation here is still wrong. Looks like you added tabs while we should keep using spaces.